db/seeds.rb

```diff
@@ -262,4 +262,40 @@ def seed_quantity(item_name, organization, storage_location, quantity)
   )
 end
 
+# Create some Vendors so Purchases can have vendor_ids
+5.times do
+  Vendor.create(
+    contact_name: Faker::FunnyName.two_word_name,
+    email: Faker::Internet.email,
+    phone: Faker::PhoneNumber.cell_phone,
+    comment: Faker::Lorem.paragraph(sentence_count: 2),
+    organization_id: Organization.all.pluck(:id).sample,
+    address: "#{Faker::Address.street_address} #{Faker::Address.city}, #{Faker::Address.state_abbr} #{Faker::Address.zip_code}",
+    business_name: Faker::Company.name,
+    latitude: rand(-90.000000000...90.000000000),
+    longitude: rand(-180.000000000...180.000000000),
+    created_at: (Date.today - rand(15).days),
+    updated_at: (Date.today - rand(15).days),
+  )
+end
+
+# Create purchases
+
+suppliers = ["Target", "Wegmans", "Walmart", "Walgreens"]
+comments = ["Maecenas ante lectus, vestibulum pellentesque arcu sed, eleifend lacinia elit. Cras accumsan varius nisl, a commodo ligula consequat nec. Aliquam tincidunt diam id placerat rutrum.", "Integer a molestie tortor. Duis pretium urna eget congue porta. Fusce aliquet dolor quis viverra volutpat.", "Nullam dictum ac lectus at scelerisque. Phasellus volutpat, sem at eleifend tristique, massa mi cursus dui, eget pharetra ligula arcu sit amet nunc."]
+
+100.times do
+  Purchase.create(
+    purchased_from: suppliers.sample,
+    comment: comments.sample,
+    organization_id: Organization.all.pluck(:id).sample,
+    storage_location_id: StorageLocation.all.pluck(:id).sample,
```

Comment:
This *could* be problematic. `StorageLocation`s are bound to a single `Organization`.

I think you could fix this by adding this block just above `Purchase.create(`
```ruby
organization = Organization.all.pluck(:id).sample
```

and then for `storage_location_id`, you can use `random_record_for_org(organization, StorageLocation)` (Ref line 237). You'll probably need to do `random_record_for_org(organization, Vendor)` for the `vendor_id` as well.